onward/conf/routes

```diff
@@ -73,3 +73,9 @@ GET        /info/tech-feedback/*path                  controllers.TechFeedbackCo
 GET        /most-read-mf2.json                        controllers.MostPopularController.renderPopularMicroformat2()
 GET        /related-mf2/*path.json                    controllers.RelatedController.renderMf2(path)
 GET        /series-mf2/*path.json                     controllers.SeriesController.renderMf2SeriesStories(path)
+
+GET        /404-test/series/experience                controllers.SeriesController.renderSeries404Test(name = "experience")
```

Comment:
Can you just do `series/$seriesName<experience|alanis|blind-date|etc>` and `enderSeries404Test(seriesName: String)` rather than repeat all these? 

Or `series/:seriesName` 

IIRC you will need to open a platform PR as you mention in your TODO :)